src/main/java/blackjack/Application.java

```diff
@@ -0,0 +1,111 @@
+package blackjack;
+
+import static blackjack.domain.Rule.DEALER_NAME;
+import static blackjack.view.InputView.inputPlayerHit;
+import static blackjack.view.InputView.inputPlayerName;
+import static blackjack.view.OutputView.printBusted;
+import static blackjack.view.OutputView.printDealerDraw;
+import static blackjack.view.OutputView.printGamblerCards;
+import static blackjack.view.OutputView.printGamblerResult;
+import static blackjack.view.OutputView.printInitialDistributionPrompt;
+import static blackjack.view.OutputView.printWinning;
+import static java.util.stream.Collectors.toList;
+
+import blackjack.domain.Round;
+import blackjack.domain.WinningDiscriminator;
+import blackjack.domain.card.Card;
+import blackjack.domain.card.CardDeck;
+import blackjack.domain.card.CardShape;
+import blackjack.domain.card.CardShuffler;
+import blackjack.domain.card.CardType;
+import blackjack.domain.gambler.Name;
+import java.util.Arrays;
+import java.util.List;
+
+public class Application {
+    public static void main(String[] args) {
+        CardDeck cardDeck = createCardDeck();
+        List<Name> playerNames = getPlayerNames();
+        Round round = new Round(cardDeck, playerNames);
+        round.distributeInitialCards();
+        printInitialDistributionPrompt(playerNames);
+        printInitialCards(round, playerNames);
+        processPlayersTurn(playerNames, round);
+        processDealerTurn(round);
+        printGameResult(round, playerNames);
+    }
+
+    private static CardDeck createCardDeck() {
+        List<Card> cards = Arrays.stream(CardShape.values())
+                .flatMap(shape -> Arrays.stream(CardType.values())
+                        .map(type -> new Card(shape, type)))
+                .collect(toList());
```

Comment:
Java 16부터 Stream에서 바로 toList() 변환이 가능합니다!
```suggestion
                .toList();
```

Reply:
네 맞습니다!
해당 방법을 알고 계시다면 문제 없을 것 같네요 👍